dev-libs/xmlsec/xmlsec-1.2.37.ebuild

```diff
@@ -36,6 +36,10 @@ BDEPEND="virtual/pkgconfig
 		)
 	)"
 
+PATCHES=(
```

Comment:
You're touching the build system in this patch => need to inherit autotools & inherit eautoreconf.

Reply:
Wrong line? The patch for 1.2.37 only touches `src/openssl/openssl_compat.h` and 1.3.0-r1 already inherits autotools and uses `eautoreconf`.

https://github.com/gentoo/gentoo/blob/b0eb525e75e849d0b9dc9bfe58d66838089dfa3e/dev-libs/xmlsec/xmlsec-1.3.0-r1.ebuild#L55

Reply:
yes, sorry!